env.sqlite2

```diff
@@ -3,3 +3,5 @@ PGID=1000
 PHP_TZ=UTC
 DB_CONNECTION=sqlite
 DB_DATABASE=/var/www/html/Lychee/nonexistent
+CACHE_DRIVER=file
+LOG_VIEWER_CACHE_DRIVER=file
```

Comment:
Pretty sure these don't belong here. These are just for testing and are Docker env, not Lychee.